vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_server_journal.go

```diff
@@ -135,6 +135,16 @@ func (a *journalArgs) Args() []string {
 // args to the provided writer, timing out at a.Timeout. If an error occurs a line
 // is written to the output.
 func (a *journalArgs) Copy(w io.Writer) {
+	// determine whether we have more than a single boot record, and if so, include the previous
+	// boot so that we always have logs from before a restart
+	args := a.Args()
+	args = append(args, "--boot")
+	if out, err := exec.Command("journalctl", "--list-boot").CombinedOutput(); err == nil && bytes.Count(bytes.TrimSpace(out), []byte("\n")) > 1 {
+		args = append(args, "-1")
+	} else {
+		args = append(args, "0")
```

Comment:
From `journalctl(1)`:

> Thus, 1 means the first boot found in the journal in chronological order, 2 the second and so on; while -0 is the last boot, -1 the boot before last, and so on. An empty offset is equivalent to specifying -0, except when the current boot is not the last boot (e.g. because `--directory` was specified to look at logs from a different machine).

So we may want `"-0"` here.